### ISSUE TITLE
default link style a missed in 2 version

### ISSUE THREAD

**Issue by jeetiss** [diegomura/react-pdf#1188]
**Describe the bug**
link renders without underline an with black color

**To Reproduce**
1. render `Link` component without overridden styles 

**Screenshots**

![links-test-js-should-render-links-as-anchors-1-diff](https://user-images.githubusercontent.com/6726016/114168698-4719f080-9939-11eb-9a2e-24bbc9bd35b9.png)

![links-test-js-should-render-links-as-links-1-diff](https://user-images.githubusercontent.com/6726016/114168707-4a14e100-9939-11eb-8201-3d4ce45cf3cd.png)

> v1 on the left side, v2 on the right

**Desktop (please complete the following information):**
 - React-pdf version v2.0.2

PS: I tried to fix this bug by myself, but this is impossible task because I don't understand [rambda](https://github.com/diegomura/react-pdf/blob/master/packages/layout/src/steps/resolveStyles.js#L19-L26
) 🤪

**Comment by diegomura** [comment]
Yes I knew about this, but wasn't sure what to do about it honestly. Not sure if applying default styles was the best, since probably people will apply their own anyways. Do you think having default ones is better?
